positron/modules/atom_browser_web_contents.js

```diff
@@ -29,6 +32,57 @@ let WebContents_prototype = {
   loadURL: function(url) {
     this._browserWindow._domWindow.location = url;
   },
+
+  openDevTools: function() {
+    // TODO: When tools can be opened inside the content window, support
+    // `detach` option to force into a new window instead.
+
+    // Ensure DevTools core modules are loaded, including support for the about
+    // URL below which is registered dynamically.
+    const { require } = Cu.import("resource://devtools/shared/Loader.jsm", {});
+    require("devtools/client/main");
```

Comment:
This will mask the global *require*, but that should be ok, since it'll only do so for this function, which doesn't need the global *require*.